api/types/container/container_wait.go

```diff
@@ -7,14 +7,6 @@ package container
 // See hack/generate-swagger-api.sh
 // ----------------------------------------------------------------------------
 
-// ContainerWaitOKBodyError container waiting error, if any
-// swagger:model ContainerWaitOKBodyError
-type ContainerWaitOKBodyError struct {
```

Comment:
I think this is just the swagger-gen randomly sorting (because there's a `map[string]` somewhere in the generator code :disappointed:

perhaps you can swap these back to their original position?